hack/make.sh

```diff
@@ -89,6 +89,11 @@ elif ${PKG_CONFIG} 'libsystemd-journal' 2> /dev/null; then
 	DOCKER_BUILDTAGS+=" journald journald_compat"
 fi
 
+# riscv64 architecture does not support CGO so disable dependencies
+if [ "$(uname -m)" = 'riscv64' ]; then
+    DOCKER_BUILDTAGS+=" exclude_disk_quota exclude_graphdriver_devicemapper"
+fi
```

Comment:
I'm not entirely convinced this is the cleanest way to handle this -- for example, we might add more bits which use `cgo` in the future, and this will then have to be manually updated.  Wouldn't it be possible to update the files in question to also check for `cgo` via build tags so that this happens automatically/transparently?  (and then if/when `riscv64` does support `cgo`, they can be more naturally supported without further changes to the explicit exclusions here)

Reply:
Addressed this on quota with cgo build tags. Found-out that devicemapper package already exists for riscv64 do no need to exclude it.

Reply:
Much cleaner and nicer this way. Thanks @tianon for pointing this out :)